plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -321,10 +322,10 @@ protected Location createMaterializedViewStorage(
         return metadataFileLocation;
     }
 
-    protected void dropMaterializedViewStorage(TrinoFileSystem fileSystem, String storageMetadataLocation)
+    protected void dropMaterializedViewStorage(ConnectorSession session, TrinoFileSystem fileSystem, String storageMetadataLocation)
```

Comment:
`TrinoFileSystemFactory` field exists in this class. Can we remove the 2nd parameter? 
```java
    protected void dropMaterializedViewStorage(ConnectorSession session, String storageMetadataLocation)
            throws IOException
    {
        TrinoFileSystem fileSystem = fileSystemFactory.create(session);
        TableMetadata metadata = TableMetadataParser.read(new ForwardingFileIo(fileSystem, isUseFileSizeFromMetadata(session)), storageMetadataLocation);
        String storageLocation = metadata.location();
        fileSystem.deleteDirectory(Location.of(storageLocation));
    }
```